### PR TITLE
Add deployment docs

### PR DESCRIPTION
**Pull request by gabrielKerekes**

### CODE REVIEW

File: docs/DEPLOYMENT.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+```
+8. Push new `master` to GitHub repo
+```
+git push origin master
+```
+9. Open Heroku logs
+```
+heroku logs --tail -a discord-soc-bot
+```
+10. Push `master` to Heroku
```
Comment (xjahic) [inline docs/DEPLOYMENT.md:36]: Maybe I would add that: "Open second terminal (keep one with the logs) and push 'master' to Heroku."

File: docs/DEPLOYMENT.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+4. Make a backup of the DB just in case anything goes wrong
+```
+pg_dump [DATABASE_URL] > dbBackup[DATE].sql
+```
+e.g.
+```
+pg_dump postgresql://discordbot:somePassword@localhost/discord_bot > dbBackup20211109.sql
+```
+5. Change `yoyo-local.ini` `database` to the `DATABASE_URL` from Heroku
+6. Run yoyo migrations
```
Comment (xjahic) [inline docs/DEPLOYMENT.md:19]: I would add - "After this command carefully check logs if everything went ok. Also connect to DB if you can and check if DB tables were updated as you wanted them"

File: docs/DEPLOYMENT.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+3. Find production `DATABASE_URL` in Heroku
+4. Make a backup of the DB just in case anything goes wrong
+```
+pg_dump [DATABASE_URL] > dbBackup[DATE].sql
+```
+e.g.
+```
+pg_dump postgresql://discordbot:somePassword@localhost/discord_bot > dbBackup20211109.sql
+```
+5. Change `yoyo-local.ini` `database` to the `DATABASE_URL` from Heroku
```
Comment (xjahic) [inline docs/DEPLOYMENT.md:18]: I would add: "If migrations weren't changed, you can skip step 5 and step 6."

**Review by xjahic — CHANGES_REQUESTED** [review]
Nice. I would add just a small details as commented.

**Comment by gabrielKerekes** [comment]
Thanks! Your suggestions should be applied. In addition I also changed all the steps to end with a `.` 🤷

**Pull request merged**